optional_plugins/robot/tests/test_robot.py

```diff
@@ -1,7 +1,7 @@
 import os
 import unittest
 
-import avocado_robot
+import avocado_robot.robot
```

Comment:
Maybe I'm just thinking about future changes, so I'm not sure if this should be part of this patch series or not (probably not). In any case, I'm wondering how the plugins' namespace will be in the future? Have you considered this?

Are all plugins going to have `avocado_*` on the package name? Or they could be part of `avocado.plugins.*`?  It feels to me that the second option it would be more explicit that we are talking about a plugin.

Reply:
@beraldoleal Right now, I'm keeping the namespace we are already using:  'avocado_$PLUGIN_NAME`. The plugins are included in the code of avocado but we're still installing them as a separate thing.

A change of namespace such as moving to `avocado.plugins.*` would need a discussion and it'll be hard not to tangle it  with the open discussion about what to do with `optional_plugins.`